app/src/main/java/org/fossasia/openevent/general/event/Event.kt

```diff
@@ -62,5 +65,8 @@ data class Event(
         val isTicketingEnabled: Boolean = false,
         val isTaxEnabled: Boolean = false,
         val isMapShown: Boolean = false,
-        val favorite: Boolean = false
+        val favorite: Boolean = false,
+        @ColumnInfo(index = true)
+        @Relationship("event-topic")
+        var eventTopic: EventTopicId? = null
```

Comment:
Why not load the entire event topic in one go?

Reply:
This way you can even show the event topic's name

Reply:
Great! I will do that :+1: